### PR TITLE
Refresh Log Collector Scripts for SSM

### LINKED ISSUE #1134: Better to keep AWS CLI version aligned
**What would you like to be added**:

Keep the AWS CLI version aligned.

**Why is this needed**:

Recently introduced AWS CLI `v2` into build scripts at #1066 and fix the path issue later on #1102.

However, there are still 2 more scripts will try to setup AWS CLI `v1` if user try to run `log-collector-script` with SSM,

* https://github.com/awslabs/amazon-eks-ami/blob/v20221112/log-collector-script/linux/eks-ssm-content.json#L27-L36
* https://github.com/awslabs/amazon-eks-ami/blob/v20221112/log-collector-script/windows/eks-ssm-content.json#L25-L34

Which might cause inconsistent AWS CLI output if there's some `$PATH` order issue.

After checking commit history, the scripts were introduced 3 years ago
* https://github.com/awslabs/amazon-eks-ami/pull/354/files

**Discussion**:

* Do we need to install AWS CLI if we had already have it installed while baking AMI? maybe we should remove it?

### PR DESCRIPTION
**Pull request by guessi**
**Issue:** 

Resolves #1134.

**Description of changes:**

* Cleanup HTML code from README.md
* Update AWS CLI version from v1 to v2
* Refresh sample command outputs for v0.7.3
* Remove dummy "PatchLinux" for Windows nodes
* Remove dummy "PatchWindows" for Linux nodes
* Rename SSM document for Linux from "EKSLogCollector" to "EKSLogCollectorLinux"
* Rename SSM document for Windows from "EKSLogCollector" to "EKSLogCollectorWindows"
* Explicitly defined permissions for run `aws ssm ...` commands

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

see comment below

*[See this guide for recommended testing for PRs.](../doc/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

**Comment by guessi** [comment]
Test with a node running with `amazon-eks-node-1.24-v20221112` AMI:

image

File: log-collector-script/linux/README.md [hunk 1]
```diff
@@ -17,93 +18,113 @@ sudo bash eks-log-collector.sh
...
 
-* Worker Node(s) should have required permissions to communicate with SSM service. IAM managed role `AmazonEC2RoleforSSM` will have all the required permission for SSM agent to run on EC2 instances. The IAM managed role `AmazonEC2RoleforSSM` has `S3:PutObject` permission to all S3 resources. 
+* Worker Node(s) should have required permissions to communicate with SSM service. IAM managed role `AmazonEC2RoleforSSM` will have all the required permission for SSM agent to run on EC2 instances. The IAM managed role `AmazonEC2RoleforSSM` has `S3:PutObject` permission to all S3 resources.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Note:* For more granular control of the IAM permission check [AWS Systems Manager Permissions link ](https://docs.aws.amazon.com/systems-manager/latest/userguide/auth-and-access-control-permissions-reference.html)
+*Note:* For more granular control of the IAM permission check [Actions defined by AWS Systems Manager](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awssystemsmanager.html%23awssystemsmanager-actions-as-permissions)
```
Comment (guessi) [inline log-collector-script/linux/README.md:96]: The link have been 302 to new address

```
$ curl --head https://docs.aws.amazon.com/systems-manager/latest/userguide/auth-and-access-control-permissions-reference.html
HTTP/1.1 302 Moved Temporarily

...
Location: https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awssystemsmanager.html%23awssystemsmanager-actions-as-permissions

...
```

File: log-collector-script/windows/README.md [hunk 2]
```diff
@@ -65,39 +70,54 @@ Archiving gathered data
...
 
-* Worker Node(s) should have required permissions to communicate with SSM service. IAM managed role `AmazonEC2RoleforSSM` will have all the required permission for SSM agent to run on EC2 instances. The IAM managed role `AmazonEC2RoleforSSM` has `S3:PutObject` permission to all S3 resources. 
+* Worker Node(s) should have required permissions to communicate with SSM service. IAM managed role `AmazonEC2RoleforSSM` will have all the required permission for SSM agent to run on EC2 instances. The IAM managed role `AmazonEC2RoleforSSM` has `S3:PutObject` permission to all S3 resources.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*Note:* For more granular control of the IAM permission check [AWS Systems Manager Permissions link ](https://docs.aws.amazon.com/systems-manager/latest/userguide/auth-and-access-control-permissions-reference.html)
+*Note:* For more granular control of the IAM permission check [Actions defined by AWS Systems Manager](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awssystemsmanager.html%23awssystemsmanager-actions-as-permissions)
```
Comment (guessi) [inline log-collector-script/windows/README.md:89]: The link have been 302 to new address

```
$ curl --head https://docs.aws.amazon.com/systems-manager/latest/userguide/auth-and-access-control-permissions-reference.html
HTTP/1.1 302 Moved Temporarily

...
Location: https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awssystemsmanager.html%23awssystemsmanager-actions-as-permissions

...
```

File: log-collector-script/linux/README.md [hunk 3]
```diff
@@ -17,93 +18,113 @@ sudo bash eks-log-collector.sh
...
+
+Trying to collect CNI configuration information...
+Trying to collect Docker daemon logs...
+Trying to Collect sandbox-image daemon information...
+Trying to Collect CPU Throttled Process Information...
+Trying to Collect IO Throttled Process Information...
+Trying to archive gathered information...
+
+        Done... your bundled logs are located in /var/log/eks_i-XXXXXXXXXXXXXXXXX_2022-12-19_1639-UTC_0.7.3.tar.gz
 ```
```
Comment (guessi) [inline log-collector-script/linux/README.md:78]: refresh sample output

**Review by cartermckinnon — APPROVED** [review]

**Pull request merged**